### PR TITLE
Update assets

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: mdn_theme/mdn/layout.html [hunk 1]
```diff
@@ -10,38 +10,37 @@
   <link href="https://developer.mozilla.org/#copyright" rel="copyright"></link>
 
   <link href="//mozorg.cdn.mozilla.net/media/css/tabzilla-min.css" rel="stylesheet" type="text/css"></link>
-      <link href="https://developer.mozilla.org/media/css/font-awesome.css?build=1419958282" media="all" rel="stylesheet"></link>
-<link href="https://developer.mozilla.org/media/css/main.css?build=1419958283" media="all" rel="stylesheet"></link>
-<link href="https://developer.mozilla.org/media/css/badges.css?build=1419958279" media="all" rel="stylesheet"></link>
+      <link href="https://developer.mozilla.org/static/styles/font-awesome.css" media="all" rel="stylesheet"></link>
+<link href="https://developer.mozilla.org/static/styles/main.css" media="all" rel="stylesheet"></link>
```
Comment (jezdez) [inline mdn_theme/mdn/layout.html:14]: I was wondering what that will mean in terms of far future headers. Are we only going to add those headers for hashed files?

Comment (openjck) [inline mdn_theme/mdn/layout.html:14]: Glad you asked. _Expires_ headers are only being changed for assets with an MD5 hash in the filename. That's what https://github.com/mozilla/kuma/pull/3640 does currently.

So this change from /media/ to /static/ isn't going to have any effect on the _Expires_ header.

Comment (jezdez) [inline mdn_theme/mdn/layout.html:14]: Ah, nice!

**Comment by jezdez** [comment]
r+

**Pull request merged**